java/openmldb-taskmanager/src/main/java/com/_4paradigm/openmldb/taskmanager/client/TaskManagerClient.java

```diff
@@ -57,72 +60,282 @@ public TaskManagerClient(String endpoint) {
         taskManagerInterface = BrpcProxy.getProxy(rpcClient, TaskManagerInterface.class);
         RpcContext.getContext().setLogId(1234L);
     }
-
     /**
      * Stop the brpc client.
      */
     public void stop() {
         rpcClient.stop();
     }
+    /**
+     * Stop the job.
+     *
+     * @param id id of job.
+     * @throws Exception
+     */
+    public void stopJob(int id) throws Exception {
+        TaskManager.StopJobRequest request = TaskManager.StopJobRequest.newBuilder()
+                .setId(id)
+                .build();
+        TaskManager.StopJobResponse response = taskManagerInterface.StopJob(request);
 
+        if (response.getCode() != 0) {
+            String errorMessage = "Fail to request, code: " + response.getCode() + ", error: " + response.getMsg();
+            throw new Exception(errorMessage);
+        }
+    }
+    /**
+     * Run sql statements in batches without default_db.
+     *
+     * @param sql query sql string.
+     * @param output_path output path.
+     * @return id of job.
+     * @throws Exception
+     */
+    public int runBatchSql(String sql, String output_path) throws Exception {
+        return runBatchSql(sql, output_path, new HashMap<String, String>(), "");
+    }
+    /**
+     * Run sql statements in batches.
+     *
+     * @param sql query sql string.
+     * @param output_path output path.
+     * @param default_db default database.
+     * @return id of job.
+     * @throws Exception
+     */
+    public int runBatchSql(String sql, String output_path, HashMap<String, String> conf, String default_db) throws Exception {
```

Comment:
The names of parameters would be `outputPath` instead `output_path`. We may change later.